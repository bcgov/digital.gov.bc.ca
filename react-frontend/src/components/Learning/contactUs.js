import React, { useContext } from 'react';
import { Col, Row } from 'react-flexbox-grid';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faEnvelope } from '@fortawesome/free-regular-svg-icons';
import Query from '../Query';
import LEARNING_ADMINS_QUERY from '../../queries/learning/learningAdmins';
import { AppConfigContext } from '../../providers/AppConfig';

import { Heading } from '../StyleComponents/headings';
import {
  BannerHorizontalContainer,
  LearningSideImageTextBox,
} from '../StyleComponents/pageContent';

function ContactUs() {
  const strapiMediaUrl =
    useContext(AppConfigContext)['state']['strapiMediaUrl'];
  return (
    <BannerHorizontalContainer>
      <Query query={LEARNING_ADMINS_QUERY}>
        {({ data: { learningAdmins } }) => {
          return (
            <Row middle="xs">
              <Col xs={12} lg={6}>
                <LearningSideImageTextBox>
                  <p>
                    Is there a course you wish we offered? Got an idea for an
                    event?
                  </p>
                  <Heading>We'd love to hear from you.</Heading>
                </LearningSideImageTextBox>
              </Col>
              {learningAdmins.map((admin, i) => {
                return (
                  <Col xs={12} sm={6} lg={3} key={i}>
                    <div
                      style={{ textAlign: 'left', marginTop: '16px' }}
                      data-testid="learning-admin"
                    >
                      <img
                        src={strapiMediaUrl + admin?.HeadShot?.url}
                        style={{
                          width: '100%',
                          borderRadius: '18px',
                          maxWidth: '300px',
                        }}
                      />
                      <p style={{ marginBottom: '0', maxWidth: '300px' }}>
                        <b>{admin.Name}</b>
                        <a
                          className="icon"
                          href={`mailto:${admin.Email}`}
                          style={{ float: 'right' }}
                        >
                          <FontAwesomeIcon
                            icon={faEnvelope}
                            style={{ color: 'black' }}
                          />
                        </a>
                      </p>
                      <p style={{ marginBottom: '0' }}>{admin.Role},</p>
                      <p style={{ marginBottom: '0' }}>{admin.Office}</p>
                    </div>
                  </Col>
                );
              })}
            </Row>
          );
        }}
      </Query>
    </BannerHorizontalContainer>
  );
}

export default ContactUs;
