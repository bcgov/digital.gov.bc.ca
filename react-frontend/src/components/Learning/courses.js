import React from 'react';
import { Col, Row } from 'react-flexbox-grid';

import { ContentBlockContainer } from '../StyleComponents/pageContent';
import { Heading } from '../StyleComponents/headings';
import EventCards from './eventCards';

// The courses are pulled from eventbrite and the content populates cards
function Courses() {
  return (
    <ContentBlockContainer id="courses">
      <Row>
        <Col sm={12}>
          <Heading>Courses</Heading>
        </Col>
      </Row>
      <EventCards isCourse={true} />
    </ContentBlockContainer>
  );
}

export default Courses;
