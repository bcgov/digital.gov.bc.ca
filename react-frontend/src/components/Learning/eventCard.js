import React from 'react';
import 'antd/dist/antd.css';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
import {
  EventCardThumnail,
  EventCardThumnailNullImg,
  EventCardTextBox,
  CardTitle,
  EventCardStyled,
} from '../StyleComponents/card';
import { HrefLinkStandalone } from '../StyleComponents/htmlTags';

function EventCard({ name, description, eventbriteUrl, coverImgSrc }) {
  return (
    <EventCardStyled>
      {coverImgSrc ? (
        <EventCardThumnail alt="" src={coverImgSrc} data-testid="thumbnail" />
      ) : (
        <EventCardThumnailNullImg />
      )}
      <EventCardTextBox>
        <CardTitle
          style={{ fontSize: '25.92px', clear: 'both' }}
          data-testid="title"
        >
          {name}
        </CardTitle>
        <p data-testid="description">{description}</p>
        <HrefLinkStandalone
          href={eventbriteUrl}
          target="_blank"
          rel="noopener noreferrer"
          style={{ marginTop: 'auto' }}
        >
          View Details &amp; Register
          <FontAwesomeIcon
            icon={faExternalLinkAlt}
            className="fa-xs"
            style={{ marginLeft: '0.3em' }}
          />
        </HrefLinkStandalone>
      </EventCardTextBox>
    </EventCardStyled>
  );
}

export default EventCard;
