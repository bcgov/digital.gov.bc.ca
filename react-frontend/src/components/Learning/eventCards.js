import React from 'react';
import { Col, Row } from 'react-flexbox-grid';
import Query from '../Query';
import EVENTS_QUERY from '../../queries/learning/events';
import EventCard from './eventCard';

/**
 * Creates a unique list of events
 * 1. without SeriesUIDs
 * 2. the soonest to start of multiple events with the same SeriesUID
 * 3. events with a SeriesUID key, but empty or null value
 *
 * @param {Object} list
 * @returns {Object}
 */
function filterBySeries(list) {
  // the events are ordered by start date in EVENTS_QUERY
  // the first event mapped will be the soonest to start.
  const seriesIDs = [];
  const fliteredList = [];

  list.map((event) => {
    if (!event.SeriesUID) {
      fliteredList.push(event);
    } else if (
      seriesIDs.indexOf(event.SeriesUID) === -1 &&
      event.SeriesUID !== false
    ) {
      seriesIDs.push(event.SeriesUID);
      fliteredList.push(event);
    }
  });
  return fliteredList;
}

function EventCards({ isCourse }) {
  return (
    <Query query={EVENTS_QUERY} isClass={isCourse}>
      {({ data: { eventbriteEvents } }) => {
        const filteredEvents = filterBySeries(eventbriteEvents);
        return (
          <Row style={{ marginBottom: '4px' }}>
            {filteredEvents.map((event, i) => {
              return (
                <Col
                  sm={12}
                  md={6}
                  lg={4}
                  key={event.EventID}
                  style={{ marginBottom: '20px' }}
                >
                  <EventCard
                    name={event?.Name}
                    description={event?.Description}
                    eventbriteUrl={event?.Url}
                    coverImgSrc={event?.ImageUrl}
                  />
                </Col>
              );
            })}
          </Row>
        );
      }}
    </Query>
  );
}

export default EventCards;
