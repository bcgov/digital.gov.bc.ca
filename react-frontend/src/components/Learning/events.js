import React from 'react';
import { Col, Row } from 'react-flexbox-grid';
import EventCards from './eventCards';

import { ContentBlockContainer } from '../StyleComponents/pageContent';
import { Heading } from '../StyleComponents/headings';
import { StyledP, HrefLinkStandalone } from '../StyleComponents/htmlTags';

function Events() {
  return (
    <ContentBlockContainer id="standards">
      <Row>
        <Col sm={12}>
          <Heading>Events</Heading>
        </Col>
      </Row>
      <EventCards isCourse={false} />
    </ContentBlockContainer>
  );
}

export default Events;
