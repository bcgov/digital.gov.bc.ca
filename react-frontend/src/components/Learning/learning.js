import React from 'react';

import ContactUs from './contactUs';
import Courses from './courses';
import Events from './events';
import Podcast from './podcast';

import DocumentTitle from 'react-document-title';
import BannerSideImage from '../PageElements/Banners/bannerSideImage';

const learningImage = require('../../images/pngIllustrations/learningWhite.png')
  .default;

function Learning() {
  return (
    <DocumentTitle title="Learning - Digital Government - Province of British Columbia">
      <div>
        <BannerSideImage
          title={'Learning'}
          content={
            'Understand and adopt new approaches to teamwork and technology.'
          }
          image={learningImage}
        />
        <Courses />
        <Events />
        <ContactUs />
        <Podcast />
      </div>
    </DocumentTitle>
  );
}

export default Learning;
