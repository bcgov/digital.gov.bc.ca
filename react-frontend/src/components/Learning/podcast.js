import React, { useContext } from 'react';
import { Col, Row } from 'react-flexbox-grid';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
import dateFormat from 'dateformat';

import PodcastBanner from './podcastBanner';
import PodcastCollapsedMenu from './podcastCollapsedMenu';

import Query from '../Query';
import PODCASTS_QUERY from '../../queries/learning/podcasts';
import { AppConfigContext } from '../../providers/AppConfig';

import { ContentBlockContainer } from '../StyleComponents/pageContent';
import { SubHeading, SubSubHeading } from '../StyleComponents/headings';
import { HrefLinkStandalone } from '../StyleComponents/htmlTags';
import { podcastUrls } from '../../constants/urls';

function Podcast() {
  const strapiMediaUrl = useContext(AppConfigContext)['state'][
    'strapiMediaUrl'
  ];
  return (
    <div>
      <PodcastBanner />
      <ContentBlockContainer id="podcasts">
        <Row>
          <Col
            style={{
              alignItems: 'baseline',
              display: 'flex',
              marginBottom: '16px',
              justifyContent: 'space-between',
            }}
            sm={12}
          >
            <SubHeading style={{ marginBottom: '0' }}>
              Recent Episodes
            </SubHeading>
            <HrefLinkStandalone
              href={podcastUrls.bcDevExchangePodcast}
              target="_blank"
              rel="noopener noreferrer"
              style={{ marginBottom: '0' }}
            >
              View all
              <FontAwesomeIcon
                icon={faExternalLinkAlt}
                style={{ paddingLeft: '5px' }}
              />
            </HrefLinkStandalone>
          </Col>
        </Row>
        {/* PODCASTS_QUERY returns the 3 most recent episodes in Strapi  */}
        <Query query={PODCASTS_QUERY}>
          {({ data: { podcasts } }) => {
            return (
              <div>
                {podcasts.map((podcast, i) => {
                  return (
                    <Row style={{ marginBottom: '16px' }} key={i}>
                      <Col xs={12} md={4}>
                        <img
                          src={strapiMediaUrl + podcast?.CoverImage?.url}
                          style={{
                            width: '100%',
                            maxHeight: '200px',
                            maxWidth: '300px',
                            objectFit: 'cover',
                          }}
                        />
                      </Col>
                      <Col xs={12} md={8}>
                        <div
                          style={{
                            display: 'flex',
                            flexDirection: 'column',
                            height: '100%',
                          }}
                        >
                          <p
                            style={{
                              color: '#606060',
                              fontSize: '13px',
                              marginBottom: '4px',
                            }}
                          >
                            {podcast.EpisodeNumber
                              ? `Episode ${podcast.EpisodeNumber} `
                              : ``}
                            &#183; {dateFormat(podcast.AirDate, 'mmmm d, yyyy')}
                          </p>
                          <SubSubHeading data-testid="podcast-title">
                            {podcast.Title}
                          </SubSubHeading>
                          <p>{podcast.Description}</p>
                          <HrefLinkStandalone
                            href={podcast.Link}
                            target="_blank"
                            rel="noopener noreferrer"
                            style={{
                              width: 'fit-content',
                              marginRight: 'auto',
                            }}
                          >
                            Listen on Libsyn
                            <FontAwesomeIcon
                              icon={faExternalLinkAlt}
                              style={{ paddingLeft: '5px' }}
                            />
                          </HrefLinkStandalone>
                        </div>
                      </Col>
                    </Row>
                  );
                })}
              </div>
            );
          }}
        </Query>
        <PodcastCollapsedMenu />
      </ContentBlockContainer>
    </div>
  );
}

export default Podcast;
