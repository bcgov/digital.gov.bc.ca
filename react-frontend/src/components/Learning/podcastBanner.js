import React from 'react';
import { Col, Row } from 'react-flexbox-grid';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faMicrophone } from '@fortawesome/free-solid-svg-icons';
import { LearningHeaderImg } from '../StyleComponents/images';
import { Heading } from '../StyleComponents/headings';
import {
  BannerHorizontalContainer,
  LearningSideImageTextBox,
} from '../StyleComponents/pageContent';

const podcastImage =
  require('../../images/pngIllustrations/podcastGray.png').default;

function PodcastBanner() {
  return (
    <BannerHorizontalContainer>
      <Row middle="xs">
        <Col xs={12} lg={6}>
          <LearningSideImageTextBox>
            <Heading>
              The Exchange Podcast <FontAwesomeIcon icon={faMicrophone} />
            </Heading>
            <p>
              Ari Hershberg talks with members of the digital government
              community.
            </p>
          </LearningSideImageTextBox>
        </Col>
        <Col xs={12} lg={6}>
          <LearningHeaderImg alt="" src={podcastImage} />
        </Col>
      </Row>
    </BannerHorizontalContainer>
  );
}

export default PodcastBanner;
