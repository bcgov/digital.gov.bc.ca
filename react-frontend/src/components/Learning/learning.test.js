import React from 'react';
import renderer from 'react-test-renderer';
import Learning from './learning';
import { MockedProvider } from '@apollo/react-testing';
import { AppConfig } from '../../providers/AppConfig';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <AppConfig>
        <MockedProvider>
          <Learning />
        </MockedProvider>
      </AppConfig>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
