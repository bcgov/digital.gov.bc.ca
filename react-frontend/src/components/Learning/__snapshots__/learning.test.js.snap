// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="sc-dIouRR sc-hHLeRK dwnlAr coahBq horizontalAlignment bannerCenterText container"
    id="main-content-anchor"
  >
    <div
      className="row middle-xs"
    >
      <div
        className="col-sm-12 col-md-6"
      >
        <div
          className="sc-eXBvqI ixCBHy sideImageText"
        >
          <h1
            className="sc-iFwKgL mHHRz bannerTitle"
          >
            Learning
          </h1>
          <div
            className="sc-iqGgem iAeeNr subTitle"
          >
             
            Understand and adopt new approaches to teamwork and technology.
             
          </div>
        </div>
      </div>
      <div
        className="col-sm-12 col-md-6"
      >
        <img
          alt=""
          className="sc-fmrZth cWMxDw sideImage"
        />
      </div>
    </div>
  </div>
  <div
    className="sc-dIouRR sc-kgflAQ dwnlAr eaTDxv horizontalAlignment contentBlockContainer container"
    id="digitalacademy"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-12"
      >
        <h3
          className="sc-hKMtZM jXKzWj subHeading"
        >
          Digital Academy LearningHub
        </h3>
        <p
          className="sc-GVOUr eBizha digitalParagraph"
        >
          Visit the 
          <a
            className="sc-gSAPjG jURRju externalLink"
            href="https://learningcentre.gww.gov.bc.ca/learninghub/learning_partner/digital-academy/"
          >
            Digital Academy LearningHub
          </a>
           page to sign-up for scheduled:
          <ul>
            <li>
              internal courses related to Agile, OpenShift, Scrum Master, and Scrum Product Owner
            </li>
            <li>
              external partner learning opportunities on a variety of topics
            </li>
          </ul>
          For more information contact 
          <a
            className="sc-gSAPjG jURRju externalLink"
            href="mailto:DigitalAcademy@gov.bc.ca"
          >
            DigitalAcademy@gov.bc.ca
          </a>
        </p>
      </div>
    </div>
  </div>
  <div
    className="sc-dIouRR sc-kgflAQ dwnlAr eaTDxv horizontalAlignment contentBlockContainer container"
    id="newsletter"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-12"
      >
        <h3
          className="sc-hKMtZM jXKzWj subHeading"
        >
          Newsletter
        </h3>
        <p
          className="sc-GVOUr eBizha digitalParagraph"
        >
          Subscribe to the 
          <a
            className="sc-gSAPjG jURRju externalLink"
            href="https://us3.list-manage.com/subscribe?u=2cd8863adab4f39ade7cbee34&id=b331c22b50"
          >
            BCDevExchange Newsletter
          </a>
           to stay up-to-date on events, learning opportunities, and what's happening at the Exchange Lab.
        </p>
      </div>
    </div>
  </div>
  <div
    className="sc-dIouRR sc-kgflAQ dwnlAr eaTDxv horizontalAlignment contentBlockContainer container"
    id="courses"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-12"
      >
        <h2
          className="sc-gsnTZi hgaMBt heading"
        >
          Courses
        </h2>
      </div>
    </div>
    <p>
      Loading...
    </p>
  </div>
  <div
    className="sc-dIouRR sc-kgflAQ dwnlAr eaTDxv horizontalAlignment contentBlockContainer container"
    id="standards"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-12"
      >
        <h2
          className="sc-gsnTZi hgaMBt heading"
        >
          Events
        </h2>
      </div>
    </div>
    <p>
      Loading...
    </p>
  </div>
  <div
    className="sc-dIouRR sc-hHLeRK dwnlAr coahBq horizontalAlignment bannerCenterText container"
    id="main-content-anchor"
  >
    <p>
      Loading...
    </p>
  </div>
  <div>
    <div
      className="sc-dIouRR sc-hHLeRK dwnlAr coahBq horizontalAlignment bannerCenterText container"
      id="main-content-anchor"
    >
      <div
        className="row middle-xs"
      >
        <div
          className="col-xs-12 col-lg-6"
        >
          <div
            className="sc-gicCDI iXnOfF"
          >
            <h2
              className="sc-gsnTZi hgaMBt heading"
            >
              The Exchange Podcast 
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-microphone "
                data-icon="microphone"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 352 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M176 352c53.02 0 96-42.98 96-96V96c0-53.02-42.98-96-96-96S80 42.98 80 96v160c0 53.02 42.98 96 96 96zm160-160h-16c-8.84 0-16 7.16-16 16v48c0 74.8-64.49 134.82-140.79 127.38C96.71 376.89 48 317.11 48 250.3V208c0-8.84-7.16-16-16-16H16c-8.84 0-16 7.16-16 16v40.16c0 89.64 63.97 169.55 152 181.69V464H96c-8.84 0-16 7.16-16 16v16c0 8.84 7.16 16 16 16h160c8.84 0 16-7.16 16-16v-16c0-8.84-7.16-16-16-16h-56v-33.77C285.71 418.47 352 344.9 352 256v-48c0-8.84-7.16-16-16-16z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </h2>
            <p>
              Ari Hershberg talks with members of the digital government community.
            </p>
          </div>
        </div>
        <div
          className="col-xs-12 col-lg-6"
        >
          <img
            alt=""
            className="sc-jmNpzm nUyQZ sideImage"
          />
        </div>
      </div>
    </div>
    <div
      className="sc-dIouRR sc-kgflAQ dwnlAr eaTDxv horizontalAlignment contentBlockContainer container"
      id="podcasts"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-12"
          style={
            Object {
              "alignItems": "baseline",
              "display": "flex",
              "justifyContent": "space-between",
              "marginBottom": "16px",
            }
          }
        >
          <h3
            className="sc-hKMtZM jXKzWj subHeading"
            style={
              Object {
                "marginBottom": "0",
              }
            }
          >
            Recent Episodes
          </h3>
          <a
            className="sc-lbxAil djTiYF externalLink"
            href="https://bcdevexchange.libsyn.com/"
            rel="noopener noreferrer"
            style={
              Object {
                "marginBottom": "0",
              }
            }
            target="_blank"
          >
            View all
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-external-link-alt fa-xs"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={
                Object {
                  "marginLeft": "0.3em",
                }
              }
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </div>
      </div>
      <p>
        Loading...
      </p>
      <div
        className="ant-collapse ant-collapse-icon-position-right"
        role={null}
        style={
          Object {
            "background": "#f2f2f2",
            "border": "none",
          }
        }
      >
        <div
          className="ant-collapse-item sc-hjriPb cDDwZC PanelStyled"
        >
          <div
            aria-expanded={false}
            className="ant-collapse-header"
            onClick={[Function]}
            onKeyPress={[Function]}
            role="button"
            tabIndex={0}
          >
            <div>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-chevron-down ant-collapse-arrow"
                data-icon="chevron-down"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={
                  Object {
                    "fontSize": "35px",
                    "paddingBottom": "6px",
                    "paddingTop": "6px",
                    "top": "12px",
                  }
                }
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </div>
            How to Subscribe
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
